src/tree.rs

```diff
@@ -38,7 +38,8 @@ lazy_static! {
     };
 }
 
-/// The root of an Orchard commitment tree.
+/// The root of an Orchard commitment tree. This must be a valid
+/// x-coordinate on the Pallas curve.
```

Comment:
Revert this, since we decided in https://github.com/zcash/zips/issues/560 that the type of an anchor will be {0 .. q<sub>P</sub>-1}, not P<sub>x</sub>.